examples/example-vue/src/Home.vue

```diff
@@ -14,21 +14,15 @@ const queryApiNames = Object.keys(client.ontology.queries);
   <div class="methods">
     <div>
       <h2>Objects ({{ objectApiNames.length }})</h2>
-      <pre v-for="objectApiName in objectApiNames" key="objectApiName">
-        client.ontology.objects.{{ objectApiName }}
-      </pre>
+      <pre v-for="objectApiName in objectApiNames" key="objectApiName">client.ontology.objects.{{ objectApiName }}</pre>
```

Comment:
The whitespace here is meaningful for vue as this is plain HTML rather than JSX where the leading and trailing whitespace in the tag would normally be stripped out